Ball_merge_surface_reconstruction/test/Ball_merge_surface_reconstruction/ball_merge_reconstruction_tests.cpp

```diff
@@ -0,0 +1,95 @@
+#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
+#include <CGAL/Ball_merge_surface_reconstruction.h>
```

Comment:
```suggestion
#include <CGAL/Ball_merge_surface_reconstruction.h>
#include <CGAL/Exact_predicates_inexact_constructions_kernel.h>
```
Change the order in tests to detect if there is the assumption that a kernel was  included.